content/docs/projects/discounts.md

```diff
@@ -0,0 +1,270 @@
+---
+title: "Discounts"
+description: "Item"
+lead: "Author/s: Celyn Raquel, Jed Reyes"
+date: 2023-07-05T14:30:18+08:00
+lastmod: 2023-07-05T14:30:18+08:00
+draft: false
+images: []
+menu:
+  docs:
+    parent: "standards"
+weight: 302
+toc: true
+---
+
+# 1 Overview
+
+Discounts are a crucial feature in purchasing and sales transactions. It enables the users to configure item rates more flexibly according to multiple factors. Discounts are maintained and are associated to the following master data DocTypes:
+
+- Item
+- Supplier
+- Customer
+
+Item discounts are discounts applied to the Price List Rate of each item in a transaction. Supplier, Customer, and Principal discounts are discounts applied to the whole transaction, distributed among the items' Rates.
+
+<img src="discounts_general_flow.png" alt="General Flow" width="100%">
```

Comment:
The image is not showing up on my end.
<img width="434" alt="image" src="https://github.com/BizKit-Tech/BizKit-Tech.github.io/assets/63850129/48e4268d-3228-4f94-a6de-57408018fa18">

I used `{{< img src="" alt="" class="border-0" >}}` for images in my page but that requires a file structure like this:
```
content/
-- docs/
---- projects/
------ discounts/
-------- index.md
-------- discounts_general_flow.png
```

Reply:
Done (f4c4dbd420dc2e4097a7f0102fc42c10aa76f42b)